app/controllers/DraftMetadataUploadController.scala

```diff
@@ -35,4 +42,30 @@ class DraftMetadataUploadController @Inject() (
       }
     }
   }
+
+  def saveDraftMetadata(consignmentId: java.util.UUID): Action[MultipartFormData[TemporaryFile]] = secureAction.async(parse.multipartFormData) {
+
+    implicit request: Request[MultipartFormData[TemporaryFile]] =>
+      val successPage = routes.DraftMetadataChecksController.draftMetadataChecksPage(consignmentId)
+      val token = request.asInstanceOf[Request[AnyContent]].token
+
+      val uploadBucket = s"${applicationConfig.draft_metadata_s3_bucket_name}-${applicationConfig.frontEndInfo.stage}"
+
+      val uploadKey = s"$consignmentId/draft-metadata.csv"
+
+      def uploadMetaData: IO[Result] = for {
+        firstFilePart <- fromOption(request.body.files.headOption)(new RuntimeException("No meta data file provided"))
```

Comment:
Abstract the error message `No metadata file provided` to a val as it is used in multiple places